packages/critters/package.json

```diff
@@ -52,6 +52,7 @@
     "domhandler": "^5.0.2",
     "htmlparser2": "^8.0.2",
     "postcss": "^8.4.23",
+    "postcss-media-query-parser": "^0.2.3",
```

Comment:
In general I would be hesitant in using a package that was last released 7 years ago.

Reply:
Makes sense. I'll try to find a different library/approach later. For now I think this covers our basic usecase.